src/validators/abstractions/base.validator.ts

```diff
@@ -0,0 +1,51 @@
+import { ObjectHasher } from "@blockr/blockr-crypto";
+import { DataAccessLayer } from "@blockr/blockr-data-access";
+import { logger } from "@blockr/blockr-logger";
+import { ValidationCondition } from "../concretes/validation.condition";
+import { IValidator } from "../interfaces/validator";
```

Comment:
Simplify import & alphabetically sort imports

```suggestion
import { IValidator } from "..";
```